addons/sale_product_configurator/static/src/js/product_list/product_list.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
```

Comment:
seems like you pushed some diff from another task/branch/PR :P

Reply:
I hate myself